lib/cosmos/tools/table_manager/table_manager.rb

```diff
@@ -739,6 +738,7 @@ def save_gui_data(name)
             else
               table.write(item.name, value)
             end
+            raise "out of range" unless item.range.include?(table.read(item.name, :RAW))
```

Comment:
I don't think everything has a range, or is a type that supports range checks.  Strings?